edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/handlers/PolicyUpdateHandler.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Hub.Core;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Config;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Responsible for listening for EdgeHub config updates (twin updates)
+    /// and pushing new authorization policy definition from EdgeHub config to the Mqtt Broker.
+    /// </summary>
+    public class PolicyUpdateHandler : IMessageProducer
```

Comment:
Will you have a separate handler for bridge config? Why not combine the two?

Reply:
I thought it better to have them separate, especially on the broker side, since we update different parts of the broker. But I might combine that on EH Core side when I work on Bridge config updates.